blackbox/docs/best_practice/sharding.txt

```diff
@@ -0,0 +1,97 @@
+.. highlight:: sh
+
+.. _sharding_guide:
+
+==============
+Sharding Guide
+==============
+
+This document is a sharding best practice guide for CrateDB.
+
+A brief recap: CrateDB tables are split into a configured number of shards, and
+then these shards are distributed across the cluster.
+
+Figuring out how many shards to use for your tables requires you to think about
+the type of data you're processing, the types of queries you're running, and
+the type of hardware you're using.
+
+.. NOTE::
+
+   This guide assumes you know the basics. If you are looking for an intro to
+   sharding, see :ref:`sql_ddl_sharding`.
+
```

Comment:
Maybe link also to 'partitoning' since each partition creates a new configured number of shards. When using partitions the # of configured shards will grow automatically. This could easily lead to over-allocation.